server/src/main/java/org/cloudfoundry/identity/uaa/scim/endpoints/ScimUserEndpoints.java

```diff
@@ -137,7 +137,7 @@ public class ScimUserEndpoints implements InitializingBean, ApplicationEventPubl
     public ScimUserEndpoints(
             final IdentityZoneManager identityZoneManager,
             final IsSelfCheck isSelfCheck,
-            final ScimUserProvisioning scimUserProvisioning,
+            final @Qualifier("scimUserProvisioning") ScimUserProvisioning scimUserProvisioning,
             final IdentityProviderProvisioning identityProviderProvisioning,
             final @Qualifier("scimUserProvisioning") ResourceMonitor<ScimUser> scimUserResourceMonitor,
```

Comment:
using 2 time same scimUserProvisioning qualifier ?

Reply:
The `JdbcScimUserProvisioning` class also implements the `ResourceManager` interface and is used as the implementation to this interface here.